src/Tasks.UnitTests/GetAssembliesMetadata_Tests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NETFRAMEWORK
```

Comment:
nit: should it rather be `FEATURE_ASSEMBLYLOADCONTEXT`?

Reply:
I don't know the difference between these two :)